ydb/docs/en/core/concepts/_includes/limits-ydb.md

```diff
@@ -43,13 +43,14 @@ The "Error type" column shows the status that the query ends with if an error oc
 
 The table below lists the limits that apply to query execution.
 
-| Parameter | Default | Explanation | Status<br/>in case of<br/>a violation<br/>of the limit |
+| Parameter | Default | Explanation | Effect<br/>in case of<br/>a violation<br/>of the limit |
 | :--- | :--- | :--- | :---: |
-| Query duration | 1800 seconds (30 minutes) | The maximum amount of time allowed for a single query to execute. | TIMEOUT |
-| Maximum number of sessions per cluster node | 1,000 | The limit on the number of sessions that clients can create with each {{ ydb-short-name }} node. | OVERLOADED |
-| Maximum query text length | 10 KB | The maximum allowable length of YQL query text. | BAD_REQUEST |
-| Maximum size of parameter values | 50 MB | The maximum total size of parameters passed when executing a previously prepared query. | BAD_REQUEST |
-| Maximum size of a row | 50 MB | The maximum total size of all fields of a single row returned or produced by the query. | PRECONDITION_FAILED |
+| Query duration | 1800 seconds (30 minutes) | The maximum amount of time allowed for a single query to execute. | Returns status code `TIMEOUT` |
+| Maximum number of sessions per cluster node | 1,000 | The limit on the number of sessions that clients can create with each {{ ydb-short-name }} node. | Returns status code `OVERLOADED` |
+| Maximum query text length | 10 KB | The maximum allowable length of YQL query text. | Returns status code `BAD_REQUEST` |
+| Maximum size of parameter values | 50 MB | The maximum total size of parameters passed when executing a previously prepared query. | Returns status code `BAD_REQUEST` |
+| Maximum size of a row | 50 MB | The maximum total size of all fields of a single row returned or produced by the query. | Returns status code `PRECONDITION_FAILED` |
+| Maximum number of [locks](../contributor/datashard-locks-and-change-visibility) per DataShard | 10,000 | The number of lock ranges per DataShard | Converts some locks to whole-shard locks, which use less memory, but lock the entire shard instead of just a part. |
```

Comment:
```suggestion
| Maximum number of [locks](../glossary.md#optimistic-locking) per DataShard | 10,000 | The number of lock ranges per DataShard | Converts some locks to whole-shard locks, which use less memory but lock the entire shard instead of just a part. |
```